src/components/PlaceThumbnail.js

```diff
@@ -23,7 +26,7 @@ class PlaceThumbnail extends React.Component {
   }
 
   render() {
-    const { coordinates, usState } = this.props
```

Comment:
Seems to be a bug related to region thumbnails

![problems-with-region-map](https://user-images.githubusercontent.com/780941/32734276-8cab4ff8-c85f-11e7-83b4-b9a42bb8e210.gif)